pkg/agent/util/net_windows.go

```diff
@@ -70,19 +80,39 @@ type Route struct {
 	RouteMetric       int
 }
 
-func (r Route) String() string {
+func (r *Route) String() string {
 	return fmt.Sprintf("LinkIndex: %d, DestinationSubnet: %s, GatewayAddress: %s, RouteMetric: %d",
 		r.LinkIndex, r.DestinationSubnet, r.GatewayAddress, r.RouteMetric)
 }
 
-func (r Route) Equal(x Route) bool {
+func (r *Route) Equal(x Route) bool {
 	return x.LinkIndex == r.LinkIndex &&
 		x.DestinationSubnet != nil &&
 		r.DestinationSubnet != nil &&
-		x.DestinationSubnet.IP.Equal(r.DestinationSubnet.IP) &&
+		x.DestinationSubnet.String() == r.DestinationSubnet.String() &&
 		x.GatewayAddress.Equal(r.GatewayAddress)
 }
 
+func (r Route) toMibIPForwardRow() *antreasyscall.MibIPForwardRow {
+	row := antreasyscall.NewIPForwardRow()
+	row.DestinationPrefix = *antreasyscall.NewAddressPrefixFromIPNet(r.DestinationSubnet)
+	row.NextHop = *antreasyscall.NewRawSockAddrInetFromIP(r.GatewayAddress)
+	row.Metric = uint32(r.RouteMetric)
+	row.Index = uint32(r.LinkIndex)
+	return row
+}
+
+func routeFromIPForwardRow(row antreasyscall.MibIPForwardRow) *Route {
```

Comment:
```suggestion
func routeFromIPForwardRow(row *antreasyscall.MibIPForwardRow) *Route {
```
To avoid copy cost given the struct contains quite some fields.